src/components/atoms/GalleryItem.js

```diff
@@ -0,0 +1,37 @@
+import styled from '@emotion/styled';
+import { tabletQuery, mobileQuery, desktopQuery, laptopQuery } from '../../utils/mediaqueries';
+
+const GalleryItem = styled.div(
+    {
+      textDecoration: 'none',
+      backgroundRepeat: 'no-repeat',
+      backgroundPosition: 'center',
+      backgroundSize: 'cover',
+  
+      [mobileQuery]: {
+        width: '300px',
+        height: '325px',
+        margin: '0 20px 20px',
+      },
+      [tabletQuery]: {
+        height: '385px',
+        width: '360px',
+        margin: '0 5px 10px',
+      },
+      [laptopQuery]: {
+        height: '405px',
+        width: '380px',
+        margin: '0 10px 20px',
+      },
+      [desktopQuery]: {
+        height: '405px',
+        width: '380px',
+        margin: '0 10px 20px',
```

Comment:
we have new types of media queries at our disposal. you have same values across laptop an desktop queries..
use the new laptopToDesktop query :)
(same goes for any other styles that are the same)